translations/en.json

```
@@ -14,5 +14,6 @@
   "waitingTime": "Waiting time",
   "worklist": "Worklist",
   "rejectOrder": "Reject Order",
-  "pickLabRequest": "Pick Lab Request"
+  "pickLabRequest": "Pick Lab Request",
+  "filter_orders_by_status": "Filter Orders by status"
```

The pattern is camelCase so this would be 
```suggestion
  "filterOrdersByStatus": "Filter Orders by status"
```